### PR TITLE
KEB Trial expiration labels

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Changes proposed in this pull request:

- add labels related to a trial expiration to the response of GetInstance endpoint

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/instance_get.go [hunk 1]
```diff
@@ -68,5 +73,16 @@ func (b *GetInstanceEndpoint) GetInstance(_ context.Context, instanceID string,
 			Labels: ResponseLabels(*op, *instance, b.config.URL, b.config.EnableKubeconfigURLLabel),
 		},
 	}
+
+	if b.config.ShowTrialExpireInfo {
+		// for test purposes on DEV env
+		if instance.GlobalAccountID == trialGAForTests && instance.SubAccountID == trialSAForTests {
```
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/broker/instance_get.go:79]: nit: the plan is to later remove this filter for a specific GA and SA? given it's behind a feature flag, I'd vote for showing when the flag is on (that way we can control whether it is on or off per env)

Alternatively, there could be one more flag of string type taking comma separated list of `GA:SA` for fine-grained control over the roll out?

**Review by wozniakjan — APPROVED** [review]
/lgtm
/approve
/hold

feel free to unhold at your convenience

File: components/kyma-environment-broker/internal/broker/instance_get.go (hunk 1 above)
Comment (szwedm) [reply components/kyma-environment-broker/internal/broker/instance_get.go:79]: GA and SA are only temporarily placed in the code, I needed to specify my own runtime to limit the change to only one instance on DEV.

**Review by wozniakjan — APPROVED** [review]
/lgtm
/approve

**Pull request merged**